apps/ledger-live-desktop/src/renderer/screens/platform/v2/Catalog/index.tsx

```diff
@@ -6,13 +6,28 @@ import { Browse } from "./Browse";
 import { useTranslation } from "react-i18next";
 import { useCatalog, useRecentlyUsedDB } from "../hooks";
 import { LocalLiveAppSection } from "./LocalLiveAppSection";
+import { useLocation } from "react-router-dom";
+import { Categories } from "@ledgerhq/live-common/wallet-api/react";
 
 export function Catalog() {
   const recentlyUsedDB = useRecentlyUsedDB();
 
   const { t } = useTranslation();
-  const { categories, recentlyUsed, disclaimer, search, localLiveApps } =
-    useCatalog(recentlyUsedDB);
+
+  const { state } = useLocation() as ReturnType<typeof useLocation> & {
+    state?: {
+      category?: Categories["selected"];
+    };
+  };
```

Comment:
Small detail but I think you should be able to use the generics here
```suggestion
  const { state } = useLocation<{
    category?: Categories["selected"];
  }>();
```